hack/util.sh

```diff
@@ -196,10 +196,10 @@ function start_os_master {
 
 	echo "[INFO] OpenShift server start at: "
 	date
-	
-	wait_for_url "${API_SCHEME}://${API_HOST}:${API_PORT}/healthz" "apiserver: " 0.25 80
-	wait_for_url "${API_SCHEME}://${API_HOST}:${API_PORT}/healthz/ready" "apiserver(ready): " 0.25 80
-	
+
+	wait_for_url "${API_SCHEME}://${API_HOST}:${API_PORT}/healthz" "apiserver: " 0.25 160
+	wait_for_url "${API_SCHEME}://${API_HOST}:${API_PORT}/healthz/ready" "apiserver(ready): " 0.25 160
```

Comment:
Health comes up after etcd, and it looks like we're ~30s to etcd start,
so...  This could be a disk issue (but this is right after the disk is
zeroed out).  Maybe we run out of IOPs cloning and then zeroing the disk?

On Fri, Feb 26, 2016 at 12:04 PM, David Eads <notifications@github.com>
wrote:

> In hack/util.sh
> <https://github.com/openshift/origin/pull/7656#discussion_r54270543>:
>
> > @@ -196,10 +196,10 @@ function start_os_master {
> >
> >  	echo "[INFO] OpenShift server start at: "
> >  	date
> > -	
> > -	wait_for_url "${API_SCHEME}://${API_HOST}:${API_PORT}/healthz" "apiserver: " 0.25 80
> > -	wait_for_url "${API_SCHEME}://${API_HOST}:${API_PORT}/healthz/ready" "apiserver(ready): " 0.25 80
> > -	
> > +
> > +	wait_for_url "${API_SCHEME}://${API_HOST}:${API_PORT}/healthz" "apiserver: " 0.25 160
> > +	wait_for_url "${API_SCHEME}://${API_HOST}:${API_PORT}/healthz/ready" "apiserver(ready): " 0.25 160
>
> and not just 40 seconds to start. 40 seconds to *API server* start.
> That's really bad.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/7656/files#r54270543>.
>